dev/README_RELEASE_HELM_CHART.md

```diff
@@ -50,6 +51,13 @@ details the steps for releasing Helm Chart.
 
 - Helm version >= 3.5.4
 
+## Build Changelog
+
+Before creating the RC, you need to build and commit the changelog for the release. For example, to list the commits:
+
+```shell
+git log --oneline helm-chart/1.1.0..main --pretty='format:- %s' -- chart/ docs/helm-chart/
+```
```

Comment:
Would be good to add a note here rhat in this case 1.1.0 is the previous version